apps/camera/index.html

```diff
@@ -10,6 +10,7 @@
     <link rel="localization" href="locales/camera.{locale}.properties">
     <link rel="localization" href="/shared/locales/date/date.{locale}.properties">
     <link rel="localization" href="/shared/locales/media/media.{locale}.properties">
+    <script defer src="/shared/js/l10n.js"></script>
```

Comment:
We were previously lazy-loading this file. Concerned this is going to add significant time to our critical-path.